pyproject.toml

```diff
@@ -0,0 +1,57 @@
+[tool.poetry]
+name = "nameko-vault"
+version = "0.1.2"
+description = "A Nameko extension to provide connection with Vault"
+authors = ["Instruct Developers <contato@instruct.com.br>"]
```

Comment:
Now we have an exclusive mail address for library related contact :raised_hands: 
Can you change the mail to "oss `at` instruct `dot` com `dot` br?